src/inference/src/os/win/win_system_conf.cpp

```diff
@@ -23,7 +23,7 @@ void CPU::init_cpu(CPU& cpu) {
         return;
     }
 
-    std::shared_ptr<char> base_shared_ptr(new char[len]);
+    std::unique_ptr<char[]> base_shared_ptr(new char[len]);
```

Comment:
I think either 
* std::shared_ptr<char[]>
* std::unique_ptr<char>

should work, shouldn't it?

Reply:
Okay, will check. Thanks!

I was thinking about this:
https://en.cppreference.com/w/cpp/memory/unique_ptr
<img width="253" alt="image" src="https://user-images.githubusercontent.com/52502732/230405738-2f06e73b-7edf-4eb1-af2d-23e6ff2d2f09.png">

Reply:
I thought by mistake that `std::unique_ptr<char>` should be correct, but I found that `std::unique_ptr<char[]>` must be here actually